src/run_benchmark_test.py

```diff
@@ -13,21 +13,31 @@
 from system import console
 from test_workflow.benchmark_test.benchmark_args import BenchmarkArgs
 from test_workflow.benchmark_test.benchmark_test_runners import BenchmarkTestRunners
+from test_workflow.benchmark_test.benchmark_test_suite import BenchmarkTestSuite
 
 
 def main() -> int:
     """
-        Entry point for Benchmark Test with bundle manifest, config file containing the required arguments for running
-        benchmarking test. Will call out in test.sh with benchmark as argument
+    Entry point for Benchmark Test with bundle manifest or for a comparison between two test executions.
+    If running a benchmark, the config file will contain the required arguments for running
+    benchmarking test. Will call out in test.sh with 'benchmark execute-test' or 'benchmark compare' as argument
     """
     benchmark_args = BenchmarkArgs()
-    console.configure(level=benchmark_args.logging_level)
-    if benchmark_args.bundle_manifest:
-        manifest: Union[BundleManifest, BuildManifest] = BundleManifest.from_file(benchmark_args.bundle_manifest) if not benchmark_args.min_distribution else \
-            BuildManifest.from_file(benchmark_args.bundle_manifest)
-        BenchmarkTestRunners.from_args(benchmark_args, manifest).run()
+
+    if benchmark_args.command == "execute-test":
+        console.configure(level=benchmark_args.logging_level)
```

Comment:
This can stay outside of the if condition as logging is required in all the cases.

Reply:
All the comments can be resolved once they have been addressed. @OVI3D0